### PR TITLE
Fix broken build

### PR DESCRIPTION
**Pull request by hooloovooo**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# EditorConfig is awesome: https://EditorConfig.org
```
Comment (OrkoHunter) [inline .editorconfig:1]: Yes!

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -23,10 +23,7 @@ jobs:
       - name: yarn install
         run: yarn install --frozen-lockfile
 
-      - name: Test
-        run: yarn test --ci --coverage
-
       - name: Publish
```
Comment (OrkoHunter) [inline .github/workflows/main.yml:26]: Should we not test before publishing? :)

**Review by OrkoHunter — APPROVED** [review]

File: .github/workflows/main.yml (hunk 2 above)
Comment (hooloovooo) [reply .github/workflows/main.yml:26]: Well. I'm not proud of this, but at the moment there aren't any tests to run.

Comment (OrkoHunter) [reply .github/workflows/main.yml:26]: Aah okay.

File: packages/embedded-techdocs/packages/app/package.json [hunk 3]
```diff
@@ -34,7 +34,7 @@
   },
   "scripts": {
     "start": "backstage-cli app:serve",
-    "build": "backstage-cli app:build",
+    "build": "backstage-cli app:build --config ../../app-config.yaml",
```
Comment (OrkoHunter) [inline packages/embedded-techdocs/packages/app/package.json:37]: Ooooh, there it is!

**Pull request merged**